src/main/java/org/gridsuite/modification/server/modifications/GenerationDispatch.java

```diff
@@ -54,6 +64,7 @@ public class GenerationDispatch extends AbstractModification {
 
     private final Map<Integer, List<Generator>> fixedSupplyGenerators = new HashMap<>();
     private final Map<Integer, List<Generator>> adjustableGenerators = new HashMap<>();
+    private final List<String> generatorsWithoutOutage = new ArrayList<>();
```

Comment:
set because we call contains a lot ?

Reply:
These data structures (Map, List) are removed in the next PR : https://github.com/gridsuite/network-modification-server/pull/247/files